$NetBSD: patch-aa,v 1.2 2025/12/11 20:57:34 vins Exp $

* The "l" modifier is unsupported as of binutils 2.36.
* Remove non standard header.

--- Makefile.orig	1996-07-10 04:00:17.000000000 +0000
+++ Makefile
@@ -115,7 +115,7 @@ PROJECTMINORRELEASE = 08
              MSUB = msub
         MKDIRHIER = mkdirhier
 
-               AR = ar clq
+               AR = ar cq
                CC = cc
               CPP = /lib/cpp $(STD_CPP_DEFINES)
                LD = ld
@@ -402,5 +402,5 @@ Makefiles::
 
 # DO NOT DELETE
 
-tokenscan.o: /usr/include/stdio.h /usr/include/sys/stdsyms.h tokenscan.h
-tstest.o: /usr/include/stdio.h /usr/include/sys/stdsyms.h tokenscan.h
+tokenscan.o: /usr/include/stdio.h tokenscan.h
+tstest.o: /usr/include/stdio.h tokenscan.h
